libs/domain/site/src/services/currency/default-currency.service.ts

```diff
@@ -11,7 +20,8 @@ export class DefaultCurrencyService implements CurrencyService {
 
   constructor(
     protected storeService = inject(StoreService),
-    protected queryService = inject(QueryService)
+    protected queryService = inject(QueryService),
+    protected localeService = inject(LocaleService)
```

Comment:
Add optional for backward compatibility with versions less then 1.2

```suggestion
    protected localeService = inject(LocaleService, null)
```